src/matching/Pattern_vs_code.ml

```diff
@@ -73,6 +74,25 @@ let hook_r2c_pro_was_here = Hook.create false
 (* Extra Helpers *)
 (*****************************************************************************)
 
+(* Scans string and tries to find the first occurrence of a substring
+ * matching a metavariable name.
+ *)
+let extract_metavariable s =
```

Comment:
I'm not sure this is needed? Couldn't we just do like we do for string literals?